lib/utils/search.js

```diff
@@ -20,29 +18,61 @@ const getSearchResults = (searchTerm, items) => {
   });
 };
 
-const getWeight = (searchTerm, words, resourceIndex, synonyms) => {
+const getWeight = (searchTerm, words, resource, resourceIndex) => {
+  let weight = 0;
+
+  weight += getFullPhraseWeight(searchTerm, resourceIndex);
+  weight += getIndividualWordsWeight(words, resourceIndex);
+  weight += getSynonymsWeight(words, resourceIndex, searchTerm);
+
+  const canReferToService = resource.referralContact || resource.referralWebsite;
+
+  if (weight > 0 && canReferToService) weight += 50;
+
+  return weight;
+};
+
+const getFullPhraseWeight = (searchTerm, resourceIndex) => {
+  if (resourceIndex.some(x => x.includes(searchTerm))) return 75;
+  return 0;
+};
+
+const getIndividualWordsWeight = (words, resourceIndex) => {
+  let wordsMatchedInTitle = words.filter(word => resourceIndex[0].includes(word)).length;
```

Comment:
As `resourceIndex={name: 'example, description: 'example'}` Would it make sense to access the title like this in case the ordering changes: `resourceIndex.name`?

Reply:
Also `const` instead of `let`

Reply:
Sorted! The resourceIndex is just a list of values as we call Object.Values when initialising it. This is precarious though and I'll look at alternatives.

Reply:
ResourceIndex issue sorted!